openstack_controller/tests/metrics.py

```diff
@@ -4548,6 +4548,18 @@
 
 
 IMAGES_METRICS_GLANCE = [
+    {
+        'name': 'openstack.glance.image.bytes',
+        'count': 1,
+        'value': 976,
```

Comment:
Where is this 976 value coming from?

Reply:
Removing openstack.glance.image.bytes from this pr